Mac App Store Signing Fails With ENOENT: no such file or directory, unlink

<!-- Which version of electron-builder are you using? -->
<!-- Please always try to use latest version before report. -->
* **Version**: 21.2.0

<!-- Which version of electron-updater are you using (if applicable)? -->

<!-- What target are you building for? -->
* **Target**: Mac App Store (MAS)

<!-- Enter your issue details below this comment. -->
<!-- If you want, you can donate to increase issue priority (https://www.electron.build/donate) -->
When I run electron-builder, and I specify that I want to create a Mac App Store version:

```
        "mac": {
            "type": "distribution",
            "target": [
                {
                    "target": "pkg"
                },
                {
                    "target": "dmg"
                },
                {
                    "target": "zip"
                },
                {
                    "target": "mas"
                }
            ],
            "hardenedRuntime": true,
            "gatekeeperAssess": false,
            "entitlements": "certs/entitlements.mac.plist",
            "entitlementsInherit": "certs/entitlements.mac.plist",
            "provisioningProfile": "embedded.provisionprofile",
            "category": "public.app-category.productivity",
            "icon": "desktop/icon.icns",
            "files": [
                "icon.icns",
                "tray.png",
                "tray@2x.png"
            ]
        }
        "mas": {
            "type": "distribution",
            "category": "public.app-category.productivity",
            "entitlements": "certs/entitlements.mas.plist",
            "entitlementsInherit": "certs/entitlements.mas.plist",
            "provisioningProfile": "embedded.provisionprofile",
            "icon": "desktop/icon.icns",
            "files": [
                "icon.icns",
                "tray.png",
                "tray@2x.png"
            ]
        },
```

The standalone mac version signs just fine. But when it gets to the MAS version, it fails with the following error:
```
• signing         file=electronbuild/mas/myapp.app identityName=Developer ID Application.......... provisioningProfile=embedded.provisionprofile
  ⨯ ENOENT: no such file or directory, unlink '/var/folders/rp/rvsxk1sx5l199p96hxtncflj2v3g7p/T/c4268fc22b00dab5be25354ae494f3b70fa01b40.plist'  stackTrace=
                                                                                                                                                   Error: ENOENT: no such file or directory, unlink '/var/folders/rp/rvsxk1sx5l199p96hxtncflj2v3g7p/T/c4268fc22b00dab5be25354ae494f3b70fa01b40.plist'
                                                                                                                                                       at runCallback (timers.js:694:18)
                                                                                                                                                       at tryOnImmediate (timers.js:665:5)
                                                                                                                                                       at processImmediate (timers.js:647:5)
                                                                                                                                                   From previous event:
                                                                                                                                                       at module.exports.getProvisioningProfileAsync (/Users/erickm/work/broadvoice.communicator/node_modules/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js:66:6)
                                                                                                                                                       at module.exports.preEmbedProvisioningProfile (/Users/johndoe/work/myapp/node_modules/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js:153:14)
                                                                                                                                                       at /Users/johndoe/work/myapp/node_modules/app-builder-lib/electron-osx-sign/sign.js:327:15
(node:16971) UnhandledPromiseRejectionWarning: Error: Command failed: codesign --sign D6BBB06A458F54EF7753EE9E20B62DA7B7B67DB6 --force --keychain /var/folders/rp/rvsxk1sx5l199p96hxtncflj2v3g7p/T/459f55100f305cd801c67e4b6f88c408f2899ac54a69abffa33deed71e0337ee.keychain --options runtime --entitlements ### certs/entitlements.mac.plist
error: The specified item could not be found in the keychain.
```

It seems to be trying to generate a temp plist file, but then the file can't be found. I tested my directory, it's not requiring sudo to create files in it. The file seems to not be created when it should be?

Have the exact same issue for the past week or so, after upgrading to `v21.2.0` from `v20.44.4`. I've also tested down until `v21.0.12` and the issue also exists in that version.

Here are some logs:

- **Version**: 21.2.0
- **Target**: DMG, MAS

```
DEBUG=off CSC_LINK=/Users/autobuild/uzee/webrtc-electron/build/tmp/osx_cert.p12 CSC_KEY_PASSWORD=password electron-builder -m --x64

  • electron-builder  version=21.2.0 os=18.7.0
  • loaded configuration  file=package.json ("build" field)
  • writing effective config  file=dist/builder-effective-config.yaml
  • unresolved deps  unresolved=deep-equalmkdirp nodeModuleDir=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules round=0
  • unresolved deps  unresolved=graceful-fs nodeModuleDir=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules round=0
  • unresolved deps  unresolved=graceful-fs nodeModuleDir=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules round=0
  • map async       taskCount=2
  • map async       taskCount=1
  • map async       taskCount=19
  • no native dependencies  
  • packaging       platform=darwin arch=x64 electron=6.0.4 appOutDir=dist/mac
  • packaging       platform=mas arch=x64 electron=6.0.4 appOutDir=dist/mas
  • map async       taskCount=2
  • map async       taskCount=1
  • map async       taskCount=2
  • map async       taskCount=1
  • map async       taskCount=228
  • map async       taskCount=334
  • map async       taskCount=8
  • map async       taskCount=8
  • path resolved   path=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/build/icon.icns outputFormat=icns
  • map async       taskCount=6
  • map async       taskCount=5
  • signing         file=dist/mas/webrtctest.app identityName=Developer ID Application: Acrobits (NH3365YY44) identityHash=5F0E57033E638E768A3A522D531B43C1CFECE33E provisioningProfile=none
  • signing         file=dist/mac/webrtctest.app identityName=Developer ID Application: Acrobits (NH3365YY44) identityHash=5F0E57033E638E768A3A522D531B43C1CFECE33E provisioningProfile=none
  • map async       taskCount=1
  • map async       taskCount=1
  • map async       taskCount=2
(node:18602) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, unlink '/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/36e25344163456a346245fd86604f27f3850af4e.plist'
    at processImmediate (internal/timers.js:439:21)
From previous event:
    at module.exports.getProvisioningProfileAsync (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js:66:6)
    at /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js:110:14
From previous event:
    at module.exports.findProvisioningProfilesAsync (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js:109:6)
    at module.exports.preEmbedProvisioningProfile (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js:162:12)
    at /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/electron-osx-sign/sign.js:327:15
    at MacPackager.sign (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/macPackager.ts:202:5)
(node:18602) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:18602) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
  • signing         file=dist/mas/webrtctest.app identityName=3rd Party Mac Developer Application: Acrobits (NH3365YY44) identityHash=09D33738334A382C7A53F999C103440C5588BB3E provisioningProfile=none
  • map async       taskCount=1
  • map async       taskCount=2
  • map async       taskCount=1
  • map async       taskCount=1
(node:18602) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 1)
  ⨯ ENOENT: no such file or directory, unlink '/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/36e25344163456a346245fd86604f27f3850af4e.plist'  stackTrace=
                                                                                                                                                   Error: ENOENT: no such file or directory, unlink '/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/36e25344163456a346245fd86604f27f3850af4e.plist'
                                                                                                                                                       at processImmediate (internal/timers.js:439:21)
                                                                                                                                                   From previous event:
                                                                                                                                                       at module.exports.getProvisioningProfileAsync (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js:66:6)
                                                                                                                                                       at /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js:110:14
                                                                                                                                                   From previous event:
                                                                                                                                                       at module.exports.findProvisioningProfilesAsync (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js:109:6)
                                                                                                                                                       at module.exports.preEmbedProvisioningProfile (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js:162:12)
                                                                                                                                                       at /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/electron-osx-sign/sign.js:327:15
                                                                                                                                                       at MacPackager.sign (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/macPackager.ts:202:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! webrtctest@0.0.0 build:osx: `DEBUG=off CSC_LINK=/Users/autobuild/uzee/webrtc-electron/build/tmp/osx_cert.p12 CSC_KEY_PASSWORD=password electron-builder -m --x64`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the webrtctest@0.0.0 build:osx script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/autobuild/.npm/_logs/2019-09-02T15_19_58_665Z-debug.log
```

Interestingly, the error changes if I use `DEBUG=electron-builder`:

```
> DEBUG=electron-builder CSC_LINK=/Users/autobuild/uzee/webrtc-electron/build/tmp/osx_cert.p12 CSC_KEY_PASSWORD=password electron-builder -m --x64

  • electron-builder  version=21.2.0 os=18.7.0
  • loaded configuration  file=package.json ("build" field)
  • effective config  config=
                        directories:
                          output: dist
                          buildResources: build
                        files:
                          - filter:
                              - '!extra/**/*'
                              - '!gulpfile.js'
                        mas:
                          entitlementsInherit: build/entitlements.mas.plist
                        afterSign: gulpfile.js
                        appx:
                          artifactName: '${productName}-${version}.${ext}'
                          publisher: '%win_publisher_name%'
                          publisherDisplayName: ''
                          backgroundColor: ''
                          identityName: ''
                        buildVersion: '1289818'
                        win:
                          target:
                            - nsis
                        productName: webrtctest
                        nsis:
                          allowToChangeInstallationDirectory: true
                          oneClick: false
                        mac:
                          category: public.app-category.business
                          gatekeeperAssess: false
                          entitlementsInherit: build/entitlements.mac.plist
                          extraResources:
                            - filter:
                                - '**/*.{arc,cache,data,win,dat,json,jpg}'
                              from: extra
                          extendInfo:
                            NSMicrophoneUsageDescription: webrtctest needs to access the microphone for outgoing audio during calls to work
                            LSMultipleInstancesProhibited: true
                            NSCameraUsageDescription: webrtctest needs to access the camera to enable video calls
                          hardenedRuntime: true
                          extraFiles:
                            - filter:
                                - '**/*.{bin,dll,lib,uz}'
                              to: ''
                              from: extra
                          binaries: []
                          type: distribution
                          target:
                            - mas
                            - dmg
                        appId: cz.acrobits.desktop.osx.test1
                        protocols:
                          - schemes:
                              - tel
                            name: webrtctest URI
                        
  • writing effective config  file=dist/builder-effective-config.yaml
  • spawning        command=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-bin/mac/app-builder node-dep-tree --dir /Users/autobuild/uzee/webrtc-electron/build/tmp/app
  • unresolved deps  unresolved=deep-equalmkdirp nodeModuleDir=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules round=0
  • unresolved deps  unresolved=graceful-fs nodeModuleDir=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules round=0
  • unresolved deps  unresolved=graceful-fs nodeModuleDir=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules round=0
  • exited          command=app-builder code=0 pid=18143 out=[{"dir":"/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules","deps":[{"name":"clone","version":"2.1.2"},{"name":"deep-equal","version":"1.1.0"},{"name":"define-properties","version":"1.1.3"},{"name":"electron-notification-shim","version":"1.1.0"},{"name":"electron-safe-ipc","version":"0.6.1"},{"name":"electron-settings","version":"3.2.0"},{"name":"electron-window-state","version":"4.1.1"},{"name":"function-bind","version":"1.1.1"},{"name":"graceful-fs","version":"4.2.2"},{"name":"has","version":"1.0.3"},{"name":"is-arguments","version":"1.0.4"},{"name":"is-date-object","version":"1.0.1"},{"name":"is-regex","version":"1.0.4"},{"name":"jsonfile","version":"4.0.0"},{"name":"minimist","version":"0.0.8"},{"name":"mkdirp","version":"0.5.1"},{"name":"object-is","version":"1.0.1"},{"name":"object-keys","version":"1.1.1"},{"name":"regexp.prototype.flags","version":"1.2.0"}]},{"dir":"/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/electron-window-state/node_modules","deps":[{"name":"jsonfile","version":"2.4.0"}]}]
  • spawning        command=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-bin/mac/app-builder rebuild-node-modules cwd=/Users/autobuild/uzee/webrtc-electron/build/tmp/app
  • map async       taskCount=2
  • map async       taskCount=19
  • map async       taskCount=1
  • no native dependencies  
  • exited          command=app-builder code=0 pid=18144
  • packaging       platform=darwin arch=x64 electron=6.0.4 appOutDir=dist/mac
  • spawning        command=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-bin/mac/app-builder unpack-electron --configuration [{"platform":"darwin","arch":"x64","version":"6.0.4"}] --output /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac --distMacOsAppName Electron.app
  • packaging       platform=mas arch=x64 electron=6.0.4 appOutDir=dist/mas
  • spawning        command=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-bin/mac/app-builder unpack-electron --configuration [{"platform":"mas","arch":"x64","version":"6.0.4"}] --output /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mas --distMacOsAppName Electron.app
  • map async       taskCount=2
  • map async       taskCount=1
  • map async       taskCount=2
  • map async       taskCount=1
  • map async       taskCount=228
  • map async       taskCount=334
  • exited          command=app-builder code=0 pid=18146
  • exited          command=app-builder code=0 pid=18145
  • async task error  error=pattern.startsWith is not a function
  ⨯ pattern.startsWith is not a function  stackTrace=
                                            TypeError: pattern.startsWith is not a function
                                                at FileMatcher.normalizePattern (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/fileMatcher.ts:58:17)
                                                at /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/fileMatcher.ts:53:54
                                                at Array.map (<anonymous>)
                                                at new FileMatcher (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/fileMatcher.ts:53:39)
                                                at computeNodeModuleFileSets (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/util/appFileCopier.ts:200:21)
                                                at /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/platformPackager.ts:279:36
                                                at async Promise.all (index 0)
                                                at AsyncTaskManager.awaitTasks (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/builder-util/src/asyncTaskManager.ts:65:25)
                                                at MacPackager.doPack (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/platformPackager.ts:208:5)
                                                at MacPackager.pack (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/macPackager.ts:118:9)
                                                at Packager.doBuild (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/packager.ts:437:9)
                                                at executeFinally (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/builder-util/src/promise.ts:12:14)
                                                at Packager._build (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/packager.ts:366:31)
                                                at Packager.build (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/packager.ts:330:12)
                                                at executeFinally (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/builder-util/src/promise.ts:12:14)
  • spawning        command=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-bin/mac/app-builder decode-plist -f /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/Electron.app/Contents/Info.plist -f /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/Electron.app/Contents/Frameworks/Electron Helper.app/Contents/Info.plist -f /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/Electron.app/Contents/Frameworks/Electron Helper EH.app/Contents/Info.plist -f /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/Electron.app/Contents/Frameworks/Electron Helper NP.app/Contents/Info.plist -f /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/Electron.app/Contents/Frameworks/Electron Helper (Renderer).app/Contents/Info.plist -f /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/Electron.app/Contents/Frameworks/Electron Helper (Plugin).app/Contents/Info.plist -f /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/Electron.app/Contents/Frameworks/Electron Helper (GPU).app/Contents/Info.plist -f /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/Electron.app/Contents/Library/LoginItems/Electron Login Helper.app/Contents/Info.plist
  • map async       taskCount=8
  • exited          command=app-builder code=0 pid=18152 out=[{"LSApplicationCategoryType":"public.app-category.developer-tools","NSHighResolutionCapable":true,"CFBundleExecutable":"Electron","CFBundleInfoDictionaryVersion":"6.0","CFBundlePackageType":"APPL","DTCompiler":"com.apple.compilers.llvm.clang.1_0","DTXcodeBuild":"9F2000","LSMinimumSystemVersion":"10.10.0","NSMainNibFile":"MainMenu","NSSupportsAutomaticGraphicsSwitching":true,"BuildMachineOSBuild":"17D102","CFBundleVersion":"6.0.4","DTSDKBuild":"10.13","DTXcode":"0941","NSPrincipalClass":"AtomApplication","CFBundleDisplayName":"Electron","CFBundleIconFile":"electron.icns","CFBundleName":"Electron","DTSDKName":"macosx10.13","CFBundleIdentifier":"com.github.Electron","CFBundleShortVersionString":"6.0.4"},{"BuildMachineOSBuild":"17D102","CFBundlePackageType":"APPL","DTXcodeBuild":"9F2000","NSSupportsAutomaticGraphicsSwitching":true,"CFBundleIdentifier":"com.github.Electron.helper","CFBundleName":"Electron Helper","DTCompiler":"com.apple.compilers.llvm.clang.1_0","DTSDKBuild":"10.13","DTSDKName":"macosx10.13","DTXcode":"0941","LSUIElement":true},null,null,{"BuildMachineOSBuild":"17D102","CFBundleName":"Electron Helper (Renderer)","DTSDKName":"macosx10.13","DTXcodeBuild":"9F2000","LSUIElement":true,"NSSupportsAutomaticGraphicsSwitching":true,"CFBundleIdentifier":"com.github.Electron.helper","CFBundlePackageType":"APPL","DTCompiler":"com.apple.compilers.llvm.clang.1_0","DTSDKBuild":"10.13","DTXcode":"0941"},{"BuildMachineOSBuild":"17D102","CFBundleName":"Electron Helper (Plugin)","DTCompiler":"com.apple.compilers.llvm.clang.1_0","DTSDKName":"macosx10.13","DTXcode":"0941","NSSupportsAutomaticGraphicsSwitching":true,"CFBundleIdentifier":"com.github.Electron.helper","CFBundlePackageType":"APPL","DTSDKBuild":"10.13","DTXcodeBuild":"9F2000","LSUIElement":true},{"NSSupportsAutomaticGraphicsSwitching":true,"CFBundleName":"Electron Helper (GPU)","CFBundlePackageType":"APPL","DTCompiler":"com.apple.compilers.llvm.clang.1_0","DTSDKBuild":"10.13","DTXcode":"0941","DTXcodeBuild":"9F2000","BuildMachineOSBuild":"17D102","CFBundleIdentifier":"com.github.Electron.helper","DTSDKName":"macosx10.13","LSUIElement":true},null]
  • spawning        command=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-bin/mac/app-builder icon --format icns --root /Users/autobuild/uzee/webrtc-electron/build/tmp/app/build --root /Users/autobuild/uzee/webrtc-electron/build/tmp/app --out /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/.icon-icns
  • path resolved   path=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/build/icon.icns outputFormat=icns
  • exited          command=app-builder code=0 pid=18153 out={"icons":[{"file":"/Users/autobuild/uzee/webrtc-electron/build/tmp/app/build/icon.icns","size":0}],"isFallback":false}
  • spawning        command=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-bin/mac/app-builder encode-plist
  • map async       taskCount=5
  • exited          command=app-builder code=0 pid=18154
  • copying files using pattern  matcher=from: /Users/autobuild/uzee/webrtc-electron/build/tmp/app/extra, to: /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/webrtctest.app/Contents/Resources, patterns: **/*.{arc,cache,data,win,dat,json,jpg}
  • copying         src=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/extra destination=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/webrtctest.app/Contents/Resources
  • copying files using pattern  matcher=from: /Users/autobuild/uzee/webrtc-electron/build/tmp/app/extra, to: /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/webrtctest.app/Contents, patterns: **/*.{bin,dll,lib,uz}
  • copying         src=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/extra destination=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/webrtctest.app/Contents
  • getPublishConfigsForUpdateInfo: no publishConfigs, detect using repository info
  • executing       file=security args=list-keychains -d user
  • executed        file=security stdout=
                          "/private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/0f452078c3dd1057fcae69307a597e5769c86c303689549eb583a344e7d9b1c1.keychain"
                          "/private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-zbODlZ/1.keychain"
                          "/private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-bUC4cG/1.keychain"
                          "/private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-wy89Mx/1.keychain"
                          "/Users/autobuild/Library/Caches/electron-builder/electron-builder-root-certs.keychain"
                          "/Users/autobuild/Library/Keychains/login.keychain-db"
                      
  • executing       file=security args=delete-keychain /var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/0f452078c3dd1057fcae69307a597e5769c86c303689549eb583a344e7d9b1c1.keychain
  • executed        file=security
  • executing       file=security args=list-keychains -d user
  • executed        file=security stdout=
                          "/private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-zbODlZ/1.keychain"
                          "/private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-bUC4cG/1.keychain"
                          "/private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-wy89Mx/1.keychain"
                          "/Users/autobuild/Library/Caches/electron-builder/electron-builder-root-certs.keychain"
                          "/Users/autobuild/Library/Keychains/login.keychain-db"
                      
  • executing       file=security args=create-keychain -p 65fbc9ebd0d7cf6d953c42a14be6808b92744716e434f3597aea7a08e11ae73e (sha256 hash) /var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/0f452078c3dd1057fcae69307a597e5769c86c303689549eb583a344e7d9b1c1.keychain
  • executed        file=security
  • executing       file=security args=unlock-keychain -p 65fbc9ebd0d7cf6d953c42a14be6808b92744716e434f3597aea7a08e11ae73e (sha256 hash) /var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/0f452078c3dd1057fcae69307a597e5769c86c303689549eb583a344e7d9b1c1.keychain
  • executed        file=security
  • executing       file=security args=set-keychain-settings /var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/0f452078c3dd1057fcae69307a597e5769c86c303689549eb583a344e7d9b1c1.keychain
  • executed        file=security
  • executing       file=security args=list-keychains -d user -s /var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/0f452078c3dd1057fcae69307a597e5769c86c303689549eb583a344e7d9b1c1.keychain /private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-zbODlZ/1.keychain /private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-bUC4cG/1.keychain /private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-wy89Mx/1.keychain /Users/autobuild/Library/Caches/electron-builder/electron-builder-root-certs.keychain /Users/autobuild/Library/Keychains/login.keychain-db
  • executed        file=security
  • executing       file=security args=import /Users/autobuild/uzee/webrtc-electron/build/tmp/osx_cert.p12 -k /var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/0f452078c3dd1057fcae69307a597e5769c86c303689549eb583a344e7d9b1c1.keychain -T /usr/bin/codesign -T /usr/bin/productbuild -P 5e884898da28047151d0e56f8dc6292773603d0d6aabbdd62a11ef721d1542d8 (sha256 hash)
  • executed        file=security stdout=
                      1 identity imported.
                      3 certificates imported.
                      
  • executing       file=security args=set-key-partition-list -S apple-tool:,apple: -s -k password /var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/0f452078c3dd1057fcae69307a597e5769c86c303689549eb583a344e7d9b1c1.keychain
  • executed        file=security stdout=
                      keychain: "/private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/0f452078c3dd1057fcae69307a597e5769c86c303689549eb583a344e7d9b1c1.keychain"
                      version: 256
                      class: 0x00000010 
                      attributes:
                          0x00000000 <uint32>=0x00000010 
                          0x00000001 <blob>=0x363739413538433300  "679A58C3\000"
                          0x00000002 <blob>=<NULL>
                          0x00000003 <uint32>=0x00000001 
                          0x00000004 <uint32>=0x00000000 
                          0x00000005 <uint32>=0x00000000 
                          0x00000006 <blob>=0x698C9990DE4E03A80F90853BF7956A6941D1F7CB  "i\214\231\220\336N\003\250\017\220\205;\367\225jiA\321\367\313"
                          0x00000007 <blob>=<NULL>
                          0x00000008 <blob>=0x7B38373139316361322D306663392D313164342D383439612D3030303530326235323132327D00  "{87191ca2-0fc9-11d4-849a-000502b52122}\000"
                          0x00000009 <uint32>=0x0000002A  "\000\000\000*"
                          0x0000000A <uint32>=0x00000800 
                          0x0000000B <uint32>=0x00000800 
                          0x0000000C <blob>=0x0000000000000000 
                          0x0000000D <blob>=0x0000000000000000 
                          0x0000000E <uint32>=0x00000001 
                          0x0000000F <uint32>=0x00000001 
                          0x00000010 <uint32>=0x00000001 
                          0x00000011 <uint32>=0x00000000 
                          0x00000012 <uint32>=0x00000001 
                          0x00000013 <uint32>=0x00000001 
                          0x00000014 <uint32>=0x00000001 
                          0x00000015 <uint32>=0x00000001 
                          0x00000016 <uint32>=0x00000001 
                          0x00000017 <uint32>=0x00000001 
                          0x00000018 <uint32>=0x00000001 
                          0x00000019 <uint32>=0x00000001 
                          0x0000001A <uint32>=0x00000001 
                      
  • executing       file=security args=find-identity -v /var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/0f452078c3dd1057fcae69307a597e5769c86c303689549eb583a344e7d9b1c1.keychain
  • executing       file=security args=find-identity -v -p 7af95cecdd0c3f541bcc912b113c3ed21ed9e4c7c32f40b73a00e2b19a9500f5 (sha256 hash) /var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/0f452078c3dd1057fcae69307a597e5769c86c303689549eb583a344e7d9b1c1.keychain
  • executed        file=security stdout=
                        1) 5F0E57033E638E768A3A522D531B43C1CFECE33E "Developer ID Application: Acrobits (NH3365YY44)"
                        2) 09D33738334A382C7A53F999C103440C5588BB3E "3rd Party Mac Developer Application: Acrobits (NH3365YY44)"
                           2 valid identities found
                      
  • executed        file=security stdout=
                        1) 5F0E57033E638E768A3A522D531B43C1CFECE33E "Developer ID Application: Acrobits (NH3365YY44)"
                        2) 2E8389AC2F20524A57D62A7C086D5A518568921B "Developer ID Installer: Acrobits (NH3365YY44)"
                        3) 09D33738334A382C7A53F999C103440C5588BB3E "3rd Party Mac Developer Application: Acrobits (NH3365YY44)"
                        4) D23833ADA814CC2B5FB44013A7D4D6DF5F87EA8E "3rd Party Mac Developer Installer: Acrobits (NH3365YY44)"
                           4 valid identities found
                      
  • signing         file=dist/mac/webrtctest.app identityName=Developer ID Application: Acrobits (NH3365YY44) identityHash=5F0E57033E638E768A3A522D531B43C1CFECE33E provisioningProfile=none
  • spawning        command=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-bin/mac/app-builder decode-plist -f /var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/36e25344163456a346245fd86604f27f3850af4e.plist
  • map async       taskCount=1
  • exited          command=app-builder code=0 pid=18169 out=[{"AppIDName":"Test WebRTC app","ApplicationIdentifierPrefix":["NH3365YY44"],"CreationDate":"2018-10-19T13:05:47Z","TeamIdentifier":["NH3365YY44"],"TimeToLive":364,"Entitlements":{"com.apple.application-identifier":"NH3365YY44.cz.acrobits.desktop.osx.test1","com.apple.developer.team-identifier":"NH3365YY44","com.apple.developer.aps-environment":"production","keychain-access-groups":["NH3365YY44.*"]},"Platform":["OSX"],"IsXcodeManaged":false,"Name":"Test WebRTC app","TeamName":"Acrobits","Version":1,"DeveloperCertificates":["MIIFozCCBIugAwIBAgIIRBA93553yNkwDQYJKoZIhvcNAQEFBQAwgZYxCzAJBgNVBAYTAlVTMRMwEQYDVQQKDApBcHBsZSBJbmMuMSwwKgYDVQQLDCNBcHBsZSBXb3JsZHdpZGUgRGV2ZWxvcGVyIFJlbGF0aW9uczFEMEIGA1UEAww7QXBwbGUgV29ybGR3aWRlIERldmVsb3BlciBSZWxhdGlvbnMgQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkwHhcNMTgxMDE5MTI1NDM1WhcNMTkxMDE5MTI1NDM1WjCBljEaMBgGCgmSJomT8ixkAQEMCk5IMzM2NVlZNDQxQzBBBgNVBAMMOjNyZCBQYXJ0eSBNYWMgRGV2ZWxvcGVyIEFwcGxpY2F0aW9uOiBBY3JvYml0cyAoTkgzMzY1WVk0NCkxEzARBgNVBAsMCk5IMzM2NVlZNDQxETAPBgNVBAoMCEFjcm9iaXRzMQswCQYDVQQGEwJDWjCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAM/9T3MeVUbr38pQ0FJm7l0d+dWI0mqNjQ9NTxo4EI1sIcdOqP848hlYOKinX5HVWQ+X04tSP8plDxmb9Z0vIXfF8AUBNv+sc4oW1AmzZXU0K2ECPpe0nubhJhqqY/e1hYSIg14nGoA9ays0355MdlksjvW0TbPmrakeiPMq3CF2NATKaEcrE/SxrHcaxOnR4Xo/bVGuNh/klBU5pJ3u2OAc1e4So7067sUgIwADfGL58DGru0MNeJljrQwPEJ7Q4RkXvXaL5xjv5A6A7+sGE7fete7t89pzYpHRmKiF504TwX8902wBsIWsP2/5aqJa9GXbBxuUZ+SU6NfmQJi15VECAwEAAaOCAfEwggHtMAwGA1UdEwEB/wQCMAAwHwYDVR0jBBgwFoAUiCcXCam2GGCL7Ou69kdZxVJUo7cwggEPBgNVHSAEggEGMIIBAjCB/wYJKoZIhvdjZAUBMIHxMIHDBggrBgEFBQcCAjCBtgyBs1JlbGlhbmNlIG9uIHRoaXMgY2VydGlmaWNhdGUgYnkgYW55IHBhcnR5IGFzc3VtZXMgYWNjZXB0YW5jZSBvZiB0aGUgdGhlbiBhcHBsaWNhYmxlIHN0YW5kYXJkIHRlcm1zIGFuZCBjb25kaXRpb25zIG9mIHVzZSwgY2VydGlmaWNhdGUgcG9saWN5IGFuZCBjZXJ0aWZpY2F0aW9uIHByYWN0aWNlIHN0YXRlbWVudHMuMCkGCCsGAQUFBwIBFh1odHRwOi8vd3d3LmFwcGxlLmNvbS9hcHBsZWNhLzAWBgNVHSUBAf8EDDAKBggrBgEFBQcDAzBNBgNVHR8ERjBEMEKgQKA+hjxodHRwOi8vZGV2ZWxvcGVyLmFwcGxlLmNvbS9jZXJ0aWZpY2F0aW9uYXV0aG9yaXR5L3d3ZHJjYS5jcmwwHQYDVR0OBBYEFGmMmZDeTgOoD5CFO/eVamlB0ffLMA4GA1UdDwEB/wQEAwIHgDATBgoqhkiG92NkBgEHAQH/BAIFADANBgkqhkiG9w0BAQUFAAOCAQEAXF0x+vzk7jW6G7AZ1Hxse5lJ+JNbyTRCTa33HMG2IAN8d6VT2eG5lxFr1Vo+4yYXqmWzFWcG7B/8Aq7Ligp6KlO2sIlaao//pqSbrYeuApUSsCDANtQuJrqJRdvb/AU3xxBLuV7AlEqTpu7LjSWBSBeolbKRTWog42mbme774nQO4BcqWIVbg7oisDlGIK4eB3RC59iKk60ZMPJV9zoHrBomrcoxLhm1UN3Sm4qajeu0rdtNsBCPVHDctv/Qi2t0mbRIgj3l5o/jUh4geTt0LrkRBlhKRhvS3b5Oqxp9NLeilzZKpf/yP5TV1D7GsD+au4Y2cYePJ9h1MwGVfJzNSg=="],"ExpirationDate":"2019-10-19T12:54:35Z","UUID":"28053fda-7e4b-421f-b99b-28252f745401"}]
  • spawning        command=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-bin/mac/app-builder decode-plist -f /Users/autobuild/uzee/webrtc-electron/build/tmp/app/build/entitlements.mac.plist -f /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/webrtctest.app/Contents/Info.plist
  • map async       taskCount=2
  • exited          command=app-builder code=0 pid=18170 out=[{"com.apple.security.device.camera":true,"com.apple.security.device.audio-input":true,"com.apple.security.cs.allow-unsigned-executable-memory":true},{"DTSDKBuild":"10.13","LSMultipleInstancesProhibited":true,"NSCameraUsageDescription":"webrtctest needs to access the camera to enable video calls","NSMicrophoneUsageDescription":"webrtctest needs to access the microphone for outgoing audio during calls to work","NSPrincipalClass":"AtomApplication","CFBundleInfoDictionaryVersion":"6.0","CFBundleURLTypes":[{"CFBundleTypeRole":"Editor","CFBundleURLName":"webrtctest URI","CFBundleURLSchemes":["tel"]}],"CFBundlePackageType":"APPL","LSMinimumSystemVersion":"10.10.0","NSAppTransportSecurity":{"NSAllowsArbitraryLoads":true,"NSAllowsLocalNetworking":true,"NSExceptionDomains":{"127.0.0.1":{"NSIncludesSubdomains":false,"NSTemporaryExceptionAllowsInsecureHTTPLoads":true,"NSTemporaryExceptionAllowsInsecureHTTPSLoads":false,"NSTemporaryExceptionMinimumTLSVersion":"1.0","NSTemporaryExceptionRequiresForwardSecrecy":false},"localhost":{"NSTemporaryExceptionAllowsInsecureHTTPSLoads":false,"NSTemporaryExceptionMinimumTLSVersion":"1.0","NSTemporaryExceptionRequiresForwardSecrecy":false,"NSIncludesSubdomains":false,"NSTemporaryExceptionAllowsInsecureHTTPLoads":true}}},"DTXcodeBuild":"9F2000","LSApplicationCategoryType":"public.app-category.business","NSHumanReadableCopyright":"Copyright © 2019 3rd Party Mac Developer Installer: Acrobits","CFBundleExecutable":"webrtctest","CFBundleVersion":"1289818","NSMainNibFile":"MainMenu","DTCompiler":"com.apple.compilers.llvm.clang.1_0","DTXcode":"0941","CFBundleIconFile":"webrtctest.icns","CFBundleShortVersionString":"0.0.0","AsarIntegrity":"{\"checksums\":{\"app.asar\":\"KJEVyVdilPhAsB9LkK1UeEL4e22RUGPiTJVLlFpMCKx0a0kWfHVulteLZJ2gTMnnvRdQs46FAUcYwa76HFcM/A==\",\"electron.asar\":\"OEdnmHGLrpEwyEdYJAizh2B/3CjoOhwzb6Cn9W/hANU2VZ2AwFGbKWXI8Mzb1+Yv+7ClwNHrnlZ8RyCBqVoqtA==\"}}","BuildMachineOSBuild":"17D102","NSSupportsAutomaticGraphicsSwitching":true,"CFBundleIdentifier":"cz.acrobits.desktop.osx.test1","CFBundleName":"webrtctest","NSHighResolutionCapable":true,"CFBundleDisplayName":"webrtctest","DTSDKName":"macosx10.13"}]
  electron-builder Error: Cannot find module 'gulpfile.js'
  electron-builder Require stack:
  electron-builder - /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/out/platformPackager.js
  electron-builder - /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/out/macPackager.js
  electron-builder - /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/out/packager.js
  electron-builder - /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/out/index.js
  electron-builder - /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/electron-builder/out/builder.js
  electron-builder - /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/electron-builder/out/cli/cli.js
  electron-builder     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:772:15)
  electron-builder     at Function.resolve (internal/modules/cjs/helpers.js:74:19)
  electron-builder     at resolveFunction (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/platformPackager.ts:646:17)
  electron-builder     at MacPackager.doPack (/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-lib/src/platformPackager.ts:245:23) +0ms
Notarization: Submitting webrtctest (REDACTED) to Apple ID "****@******.cz" under team "*******" using password "*******************"
This process can take up to 1 Hour...
Elapsed time: 5m, 16s 729.135376ms
  • building        target=DMG arch=x64 file=dist/webrtctest-0.0.0.dmg
  • spawning        command=hdiutil create -srcfolder /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/mac/webrtctest.app -volname webrtctest 0.0.0 -anyowners -nospotlight -format UDRW -quiet -fs HFS+ -fsargs -c c=64,a=16,e=16 /private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-nwdrlN/1.dmg
  • exited          command=hdiutil code=0 pid=18298
  • executing       file=hdiutil args=resize -size 163752561 /private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-nwdrlN/1.dmg
  • executed        file=hdiutil
  • executing       file=hdiutil args=attach -noverify -noautoopen -readwrite /private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-nwdrlN/1.dmg
  • executed        file=hdiutil stdout=
                      /dev/disk2          	GUID_partition_scheme          	
                      /dev/disk2s1        	Apple_HFS                      	/Volumes/webrtctest 0.0.0
                      
  • spawning        command=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-bin/mac/app-builder dmg --volume /Volumes/webrtctest 0.0.0 --icon /Users/autobuild/uzee/webrtc-electron/build/tmp/app/build/icon.icns --background /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/dmg-builder/templates/background.tiff
  • copy files      from=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/build/icon.icns to=/Volumes/webrtctest 0.0.0/.VolumeIcon.icns isUseHardLinks=false
  • copy files      from=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/dmg-builder/templates/background.tiff
                    to=/Volumes/webrtctest 0.0.0/.background/background.tiff
                    isUseHardLinks=false
  • execute command  command=sips -g pixelHeight -g pixelWidth /Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/dmg-builder/templates/background.tiff workingDirectory=
  • command executed  executable=sips out=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/dmg-builder/templates/background.tiff
  pixelHeight: 380
  pixelWidth: 540

  • exited          command=app-builder code=0 pid=18340 out={"backgroundWidth":540,"backgroundHeight":380,"backgroundFile":"/Volumes/webrtctest 0.0.0/.background/background.tiff"}
  • executing       file=ln args=-s /Applications /Volumes/webrtctest 0.0.0/Applications
  • executed        file=ln
  • executing       file=/usr/bin/python args=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/dmg-builder/vendor/dmgbuild/core.py cwd=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/dmg-builder/vendor env=
                      {
                        "volumePath": "/Volumes/webrtctest 0.0.0",
                        "appFileName": "webrtctest.app",
                        "iconSize": 80,
                        "iconTextSize": 12,
                        "PYTHONIOENCODING": "utf8",
                        "windowWidth": "540",
                        "windowHeight": "380",
                        "windowX": 400,
                        "windowY": "530",
                        "backgroundWidth": 540,
                        "backgroundHeight": 380,
                        "backgroundFile": "/Volumes/webrtctest 0.0.0/.background/background.tiff",
                        "iconLocations": "'webrtctest.app': (130, 220),\n'Applications': (410, 220)"
                      }
  • executed        file=/usr/bin/python
  • executing       file=hdiutil args=detach -quiet /dev/disk2
  • executed        file=hdiutil
  • spawning        command=hdiutil convert /private/var/folders/bj/1djgpnw93x5103hd2gqq6_5m0000gn/T/t-nwdrlN/1.dmg -ov -format UDZO -o /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/webrtctest-0.0.0.dmg -imagekey zlib-level=9 -quiet
  • exited          command=hdiutil code=0 pid=18369
  • building block map  blockMapFile=dist/webrtctest-0.0.0.dmg.blockmap
  • spawning        command=/Users/autobuild/uzee/webrtc-electron/build/tmp/app/node_modules/app-builder-bin/mac/app-builder blockmap --input /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/webrtctest-0.0.0.dmg --output /Users/autobuild/uzee/webrtc-electron/build/tmp/app/dist/webrtctest-0.0.0.dmg.blockmap
  • exited          command=app-builder code=0 pid=18371 out={"size":67058633,"sha512":"sjxTAQExJgnCG/J6DiR3pK7U6zVJsd7uoogAoHjTBMtXkDTFnqv2mlg4lqdEwOOvw6Pqe6ljHhFhSLVneXiI9g=="}
  • getPublishConfigsForUpdateInfo: no publishConfigs, detect using repository info
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! webrtctest@0.0.0 build:osx: `DEBUG=electron-builder CSC_LINK=/Users/autobuild/uzee/webrtc-electron/build/tmp/osx_cert.p12 CSC_KEY_PASSWORD=password electron-builder -m --x64`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the webrtctest@0.0.0 build:osx script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/autobuild/.npm/_logs/2019-09-02T15_04_21_504Z-debug.log
```

Only building DMG target without Mac App Store works fine.

--- a/packages/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js
+++ b/packages/app-builder-lib/electron-osx-sign/util-provisioning-profiles.js
@@ -64,8 +64,13 @@ var getProvisioningProfileAsync = module.exports.getProvisioningProfileAsync = f
     '-i', filePath // Use infile as source of data
   ])
     .then(async function (result) {
+      // make filename unique so it doesn't create issues with parallel method calls
+      const timestamp = process.hrtime.bigint
+        ? process.hrtime.bigint().toString()
+        : process.hrtime().join('')
+
       // todo read directly
-      const tempFile = path.join(os.tmpdir(), `${require('crypto').createHash('sha1').update(filePath).digest("hex")}.plist`)
+      const tempFile = path.join(os.tmpdir(), `${require('crypto').createHash('sha1').update(filePath).update(timestamp).digest('hex')}.plist`)
       await fs.outputFile(tempFile, result)
       const plistContent = await executeAppBuilderAsJson(["decode-plist", "-f", tempFile])
       await fs.unlink(tempFile)

